components/Cards.js

```diff
@@ -1,3 +1,45 @@
+const { default: Axios } = require("axios");
+
+axios.get("https://lambda-times-api.herokuapp.com/articles").then(response => {
+    const articles = response.data.articles;
+
+    const articlesArray = Object.keys(articles).map(a => a);
+
+    const cardsContainer = document.querySelector("div.cards-container");
+    
+    articlesArray.forEach(language => {
+        articles[language].forEach(articleObj => {
+            function articleMaker (article) {
```

Comment:
Would recommend that you pull this function definition outside of the double forEach loop. This is just a general rule of thumb to avoid issues like this https://rules.sonarsource.com/javascript/RSPEC-1515